services/slack/intermediate.go

```diff
@@ -405,6 +418,10 @@ func (t *Transformer) TransformPosts(slackExport *SlackExport, attachmentsDir st
 						}
 					} else if post.Files != nil {
 						for _, file := range post.Files {
+							if file.Name == "" {
+								t.Logger.Warnf("Not able to access file %s as file access is denied so skipping", file.Id)
```

Comment:
Are you sure that the only reason `file.Name` is empty is that file access is denied? Can't there by any other reason?

Reply:
i saw all the other attachments and in slack if you upload a file with empty name slack will give it a default name of its extension so like if the file is .pdf then file name will pdf so it cannot be empty in any other case